exchange_server/assets/dashboards/overview.json

```diff
@@ -0,0 +1,799 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Exchange Server Overview",
+    "read_only": false,
+    "description": null,
```

Comment:
This is usually filled out. Example here: https://github.com/DataDog/integrations-core/pull/6806/files#diff-423d06330dedae5497365bc3cb02bcbfR18